tests/subsys/bluetooth/fast_pair/crypto/Kconfig

```diff
@@ -4,6 +4,31 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+config FP_CRYPTO_TEST_PSA
```

Comment:
Can we use the same approach for naming test options that we adopted here https://github.com/kapi-no/sdk-nrf-google-fast-pair/blob/google_finder_extension/tests/subsys/bluetooth/fast_pair/storage/factory_reset/Kconfig?

```
config TEST_BT_FAST_PAIR_CRYPTO_PSA
```
This clearly shows the relation between these configuration options

Could you also select the proper option configuration instead of default overrides?

Reply:
Ok, will do

Reply:
> Could you also select the proper option configuration instead of default overrides?

That would require introducing board-specific Kconfig overlays (e.g. `CONFIG_BUILD_WITH_TFM` only needs to be enabled for `_ns` targets). I tried to avoid it to let users run the test on any nRF board.

Reply:
> Could you also select the proper option configuration instead of default overrides?

TF-M build type is a choice. We cannot use select here.